### PR TITLE
*: add more daily benchmark test cases

### PR DESCRIPTION
**Pull request by tiancaiamao**
### What problem does this PR solve?

Problem Summary:

### What is changed and how it works?

What's Changed:

- move some code to the util/benchdaily package
- make more bench test cases run daily
- update the Makefile 

How it Works:

There are some `bench_test.go` files in different directories, and a `TestBenchDaily` function.
The `bench-daily` rule in the Makefile will run test `TestBenchDaily` function and generate the json file,
and then those json file are combined to generate the finally json file.

### Check List 

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)

make daily-bench TO=xxx.json

- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note 

```release-note
None
```

### CODE REVIEW

**Comment by tiancaiamao** [comment]
ref https://github.com/pingcap/tidb/issues/25754

**Review by xhebox — APPROVED** [review]
One suggestion, maybe `github.com/pingcap/tidb/benchkit`, just like the new `testkit`? I think there are too many packages in the util, and nobody really cares except the author.

File: .gitignore [hunk 1]
```diff
@@ -19,3 +19,4 @@ vendor
 /_tools/
 .DS_Store
 .vscode
+bench_daily.json
```
Comment (tangenta) [inline .gitignore:22]:
```suggestion
bench_daily.json

```

**Review by tangenta — APPROVED** [review]

**Pull request merged**